doc/advanced_configuration.rst

```diff
@@ -24,6 +24,43 @@ the sphinx configuration dictionary:
 
 Keep in mind that both lists have to be of the same length.
 
+.. _build_pattern:
+
+Building examples matching a pattern
+====================================
+
+By default, Sphinx-Gallery execute only examples beginning with ``plot``. However,
+if this naming convention does not suit your project, you can modify this pattern
+in your Sphinx ``conf.py``. For example:
+
+.. code-block:: python
+
+    sphinx_gallery_conf = {
+        'filename_pattern'  : '/plot_compute_'
+    }
+
+will build all examples starting with ``plot_compute_``. This is also useful if you
+want to build only a subset of the examples, or skip some examples. For example, to
+skip building examples starting with ``plot_long_examples_``, you would do:
+
+.. code-block:: python
+
+    sphinx_gallery_conf = {
+        'filename_pattern' : '^(?!/plot_long_examples_).+'
```

Comment:
Not sure that works as you intended. In any case I would put the simple usage first (i.e. plot_awesome_example.py') below. You need to escape the dot too if you want to be really picky.